### PR TITLE
[feat] added cart checkout page

### PR DESCRIPTION
**Pull request by saurabhan**
added cart checkout page and fixed nav links

### CODE REVIEW

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                    
+                </div>
+                <h5>XL</h5>
+                <h5>4</h5>
+                <h5>Rs. 1599</h5>
+                <span>x</span>
+            </div>
+            <div class="cart-header">
+                <div>
+                    <img class="cart-img" src="../Assets/men-prod4.jpg" alt="">
```
Comment (purvasharmadev) [inline pages/cart.html]: add alt for images.

**Review by purvasharmadev — APPROVED** [review]
everything looks good to go. Nice work!

File: main.css [hunk 2]
```diff
@@ -296,3 +297,103 @@ a{
...
+}
+.checkout-button{
+    background-color: var(--colors-black);
+    color: var(--font-white);
+    padding: var(--space-10);
+    border: none;
+    font-size: large;
+    letter-spacing: var(--letterSpacings-wider);
+    transition: var(--transitions-default);
+}
```
Comment (abhishektanwar) [inline main.css]: normal pointer is shown on the checkout button . Please added cursor here.

**Review by abhishektanwar — APPROVED** [review]
Code is clean and readable. Good to merge after working on the suggestion.

File: main.css (hunk 2 above)
Comment (saurabhan) [reply main.css]: fixed, thank you

File: pages/cart.html (hunk 1 above)
Comment (saurabhan) [reply pages/cart.html]: fixed, thanks

**Pull request merged**